.swiftlint.yml

```diff
@@ -1,12 +1,28 @@
+parent_config: https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml
```

Comment:
I know we bounced this idea, but @deepesh-vasthimal-cko do we want to clone whatever RayWenderlich does?

I have nothing against copying those rules for ourselves or discussing / toggling as we find useful, but do we really want to create this dependency?